components/admin-panel/sections/ActivityLog/index.js

```diff
@@ -88,12 +89,18 @@ const ActivityLog = ({ accountSlug }) => {
     });
   };
 
+  if (error) {
+    return <MessageBoxGraphqlError error={error} />;
+  }
+
```

Comment:
We should not block the UI if there's an error, people should be able to change their filters

Reply:
This is a very simple fix I felt so I've move the error within the filters. Now the error should appear along with the filters 👍🏽